cadctap/cadctap/tests/data/help_create.txt

```diff
@@ -1,6 +1,7 @@
-usage: cadc-tap create [-h] [-d | -q | -v] [--host HOST]
-                       [--cert CERT | -n | --netrc-file NETRC_FILE | -u USER | -a | --anon]
-                       [-f {VOSITable,VOTable}] [-s SERVICE]
+usage: cadc-tap create [-h]
+                       [--cert CERT | -n | --netrc-file NETRC_FILE | -u USER | -a]
+                       [--host HOST] [-d | -q | -v] [-f {VOSITable,VOTable}]
+                       [-s SERVICE]
                        TABLENAME TABLEDEFINITION
 
 Create a table
```

Comment:
Please add a note to each command explaining how the authentication discovery works so that the users are informed.

Reply:
Done.